oxidation/libparsec/crates/protocol/tests/test_vlob.rs

```diff
@@ -399,7 +429,37 @@ fn serde_vlob_read_rep(#[case] raw_expected: (&[u8], authenticated_cmds::vlob_re
         )
     )
 )]
-#[case::with_sequester_blob(
+#[case::without(
```

Comment:
```suggestion
#[case::without_sequester_blob(
```

Reply:
We shouldn't have a case for every field which can be nullable

Reply:
🤷 the difference between `full` and `without` is the field `sequester_blob` that is set to `None` or something.

Reply:
But I prefer if it stays ISO with others with many fields set to `None`